lacework/alert_channel.go

```diff
@@ -0,0 +1,18 @@
+package lacework
+
+import (
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
```

Comment:
nit: we use `goimports` to order these imports
```suggestion
	"github.com/pkg/errors"

	"github.com/lacework/go-sdk/api"
```